apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -39,7 +39,7 @@ export const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
   start_new_zone: z.boolean(),
   zone_id: IdSchema,
   zone_title: z.string().nullable(),
-  question_title: z.string(),
+  question_title: z.string().nullable(),
```

Comment:
Nit, consider pulling this (both here and elsewhere) from `QuestionSchema.shape`.